### PR TITLE
feat: [DAT-488] Upgrade plan form - Add confirmation message

### PR DESCRIPTION
**Pull request by nreal**
# Background
We add a new modal when the user upgrades a new plan 
Example:
Capture2

### CODE REVIEW

File: src/components/UpgradePlanForm/ConfirmationMessage.test.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    const { getByText } = render(
+      <IntlProvider>
+        <ConfirmationMessage isSubscriber={true} amountSubscribers={1}></ConfirmationMessage>
+      </IntlProvider>,
+    );
+
+    // Assert
+    expect(getByText('upgradePlanForm.confirmation_title')).toBeInTheDocument();
+    expect(getByText('upgradePlanForm.confirmation_subtitle_contact')).toBeInTheDocument();
+  });
```
Comment (cbernat) [inline src/components/UpgradePlanForm/ConfirmationMessage.test.js:32]: This test is duplicated 🤔 You must've meaning to add a test for non subscriber account right?

File: src/components/UpgradePlanForm/UpgradePlanForm.js [hunk 2]
```diff
@@ -78,7 +91,7 @@ const UpgradePlanForm = ({
     return errors;
   };
 
-  return !state.isLoading ? (
+  return !state.isLoading && !confirmationMessage ? (
```
Comment (cbernat) [inline src/components/UpgradePlanForm/UpgradePlanForm.js:94]: How about having this logic?
```
return state.isLoading? (<Loading>):
  (confirmationMessage? (
    <ConfirmationMessage
      isSubscriber={isSubscriber}
      amountSubscribers={amountSubscribers}
    ></ConfirmationMessage>
  ): 
  .... {/*The rest of the code here*/}
  )

File: src/components/UpgradePlanForm/ConfirmationMessage.test.js (hunk 1 above)
Comment (nreal) [reply src/components/UpgradePlanForm/ConfirmationMessage.test.js]: Yes, I changed this test for non subscribers accounts, done

File: src/components/UpgradePlanForm/UpgradePlanForm.js (hunk 2 above)
Comment (nreal) [reply src/components/UpgradePlanForm/UpgradePlanForm.js]: yes is working, done

**Comment by cbernat** [comment]
Very nice @nreal ! One minor detail, could you merge the commits into one? We try to make the commit messages to be readable in the release notes.
Instead of this: 

image
 
 How about fixing the commit message to something like this:
`feat(upgrade-plan): add confirmation message for upgrade popup`

**Comment by cbernat** [comment]
🚀

**Pull request merged**